addons/payment/models/payment_acquirer.py

```diff
@@ -100,21 +111,29 @@ class PaymentAcquirer(models.Model):
     post_msg = fields.Html(
         'Thanks Message', translate=True,
         help='Message displayed after having done the payment process.')
+    has_post_msg = fields.Boolean(compute="_has_post_msg")
```

Comment:
because the messages are HTML strings they have HTML tags like <br> that are also there if the variable is not set. I first did the check in the template with length > 11 but that did not always give the wanted outcome.
The regexp eliminates all HTML tags and checks if there is an actual string.

Reply:
The issue is that the html editor is adding html tags for no reasons. This should be fixed instead of trying to hack the issue.

Reply:
To check with @Gorash